ramalama/annotations.py

```diff
@@ -1,4 +1,5 @@
-# These annotations are based off the proposed standard:
+# These annotations are based off the standards:
+# https://github.com/opencontainers/image-spec/blob/main/specs-go/v1/annotations.go
```

Comment:
Is this supposed to be referring to the Go sources, or the spec itself, which is at https://github.com/opencontainers/image-spec/blob/main/annotations.md?

Reply:
I was looking at code, found it more readable but you are correct, should link to actual doc and not code implementation. Fixed.